src/devices/Mbi5027/Mbi5027PinMapping.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device.Multiplexing
+{
+    /// <summary>
+    /// Represents pin bindings for the MBI5027.
+    /// </summary>
+    public struct Mbi5027PinMapping
+    {
+        /// <param name="sdi">Serial data in pin</param>
+        /// <param name="clk">Shift register pin (shift to data register)</param>
+        /// <param name="le">Register clock pin (latch)</param>
```

Comment:
should we call it just `latch` or `latchEnabled` (same for oe -> `outputEnabled`)

Reply:
I stuck with the names specified in the datasheet. That seemed the most intuitive/sensible to me. I picked better names in the `ShiftRegister` samples, since there is no datasheet for it. I'll update that one to use `LatchEnable`, which I agree is better.

Reply:
Here are the two data sheets I was targeting. Looking at them makes it more obvious why I targeted the specific pin names.

MBI5027
![image](https://user-images.githubusercontent.com/2608468/93821071-e6cc9700-fc12-11ea-8888-ce92b79edc20.png)

SN74HC595
![image](https://user-images.githubusercontent.com/2608468/93821143-0663bf80-fc13-11ea-838f-4045d03b5afa.png)